crates/sparrow-materialize/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod materialize;
```

Comment:
Why is this a public module and not just stored here? This means that logically, people use `sparrow_materialize::materialize::Materialization`, which seems to stutter (eg., what else is going to be in `sparrow_materialize` that isn't also in `materialize`)?

We could maybe have a `materialization.rs` and a `materialization_control.rs` and then expose both of them with `pub use`? Other thoughts?